src/object/fallblock.hpp

```diff
@@ -28,18 +28,20 @@ class FallBlock : public MovingSprite
 public:
   FallBlock(const ReaderMapping& reader);
   
-  virtual void update(float dt_sec) override;
-
+  virtual void update(float dt_sec) override;
+
   virtual HitResponse collision(GameObject& other, const CollisionHit& hit) override;
-  virtual void collision_solid(const CollisionHit& hit) override;
-
+  virtual void collision_solid(const CollisionHit& hit) override;
+
   virtual void draw(DrawingContext& context) override;
   
   virtual std::string get_class() const override { return "fallblock"; }
   virtual std::string get_display_name() const override { return _("Falling Platform"); }
+
+  virtual void on_flip(float height) override;
   
 protected:
-  enum State
+  enum State
   {

```

Comment:
...what changed on that line...? o.O

<sub>(I hope it's not an LF changed to a CRLF...)</sub>

Reply:
It's probably the other way round. I doubt gedit, on Linux, would put CRs there.